RunNPT.ps1

```diff
@@ -88,7 +88,7 @@ Param(
     [string] $Config = "Default",
 
     [parameter(Mandatory=$false)]  
-    [Array] $ToolList = @("ntttcp", "latte", "cps", "ctstraffic"),
+    [Array] $ToolList = @("ntttcp", "latte", "cps", "ctstraffic", "ncps"),
```

Comment:
why no secnetperf and l4ping here?

Reply:
whoops, forgot about this script as I do not use it often, will add.